src/config/FirebaseConfig.jsx

```diff
@@ -1,6 +1,7 @@
 export const firebaseConfig = {
   apiKey: 'AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA',
   authDomain: 'dreamerscholars.firebaseapp.com',
+  projectId: 'dreamerscholars',
```

Comment:
This change is related to talking to Firestore.

Reply:
Which means it's not needed here?

Reply:
I think this file is need in order for the `LoginButton` component to work